src/main/java/com/gmail/nossr50/config/skills/repair/RepairConfig.java

```diff
@@ -26,6 +27,11 @@ public RepairConfig(String fileName) {
     protected void loadKeys() {
         repairables = new ArrayList<Repairable>();
 
+        if (!config.isConfigurationSection("Repairables")) {
+            mcMMO.p.getLogger().severe("Could not find Repairables section in " + fileName);
+            return;
+        }
+
```

Comment:
Technically for a micro-optimization you could move this above the creation of the empty array list object